rustls/src/msgs/macros.rs

```diff
@@ -56,8 +56,8 @@ macro_rules! enum_builder {
         impl From<$uint> for $enum_name {
             fn from(x: $uint) -> Self {
                 match x {
-                    $($enum_val => $enum_name::$enum_var),*
-                    $(, $($enum_val_nd => $enum_name::$enum_var_nd),* )?
+                    $($enum_value_dbg_fmt => $enum_name::$enum_var_dbg_fmt),*
+                    $(, $($enum_value_no_fmt => $enum_name::$enum_var_no_fmt),* )?
```

Comment:
I would also like to improve testing of this match arm to keep Codecov happier and improve testing of this in general.